Traceback (most recent call last):
  File "C:\Users\ckeulen\anaconda3\Lib\site-packages\jupyter_cache\executors\utils.py", line 58, in single_nb_execution
    executenb(
  File "C:\Users\ckeulen\anaconda3\Lib\site-packages\nbclient\client.py", line 1305, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\ckeulen\anaconda3\Lib\site-packages\jupyter_core\utils\__init__.py", line 173, in wrapped
    return loop.run_until_complete(inner)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\ckeulen\anaconda3\Lib\asyncio\base_events.py", line 653, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "C:\Users\ckeulen\anaconda3\Lib\site-packages\nbclient\client.py", line 705, in async_execute
    await self.async_execute_cell(
  File "C:\Users\ckeulen\anaconda3\Lib\site-packages\nbclient\client.py", line 1058, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "C:\Users\ckeulen\anaconda3\Lib\site-packages\nbclient\client.py", line 914, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
#import librarier
import CoolProp.CoolProp as CP

#define variables
V_A = 2   #tank A volume in m3
V_B = 3   #tank B volume in m3
m_A = 500   #fluid in tank A mass in kg
m_B = 0.25   #fluid in tank B mass in kg

T = 25 + 273.15   #temperature in K

D_A = m_A / V_A   #density of tank A in kg/m3 used in CoolProp
D_B = m_B / V_B   #density of tank B in kg/m3 used in CoolProp

fluid = "water"  # define the fluid or material of interest

x_A = CP.PropsSI("Q", "T", T, "D", D_A , fluid)   #mixture quality in A
x_B = CP.PropsSI("Q", "T", T, "D", D_B , fluid)   #mixture quality in B

m_l_A = (1 - x_A) * m_A   #liquid mass in A in kg
m_l_B = (1 - x_B) * m_B   #liquid mass in B in kg

print('liquid mass in A is', f"{m_l_A:.2f}", 'kg')
print('liquid mass in B is', f"{m_l_B:.2f}", 'kg')

P_A = CP.PropsSI("P", "T", T, "D", D_A , fluid)   #pressure in A in Pa
P_B = CP.PropsSI("P", "T", T, "D", D_B , fluid)   #pressure in B in Pa

print('pressure in A is', f"{P_A:.2f}", 'Pa')
print('pressure in B is', f"{P_B:.2f}", 'Pa')
------------------


[1;31m---------------------------------------------------------------------------[0m
[1;31mModuleNotFoundError[0m                       Traceback (most recent call last)
Cell [1;32mIn[1], line 2[0m
[0;32m      1[0m [38;5;66;03m#import librarier[39;00m
[1;32m----> 2[0m [38;5;28;01mimport[39;00m [38;5;21;01mCoolProp[39;00m[38;5;21;01m.[39;00m[38;5;21;01mCoolProp[39;00m [38;5;28;01mas[39;00m [38;5;21;01mCP[39;00m
[0;32m      4[0m [38;5;66;03m#define variables[39;00m
[0;32m      5[0m V_A [38;5;241m=[39m [38;5;241m2[39m   [38;5;66;03m#tank A volume in m3[39;00m

[1;31mModuleNotFoundError[0m: No module named 'CoolProp'

